pyracf/common/add_operation_error.py

```diff
@@ -0,0 +1,24 @@
+"""Exception to use when Add operation would alter an existing profile."""
+
+
+class AddOperationError(Exception):
+    """
+    Raised when a profile passed into an Add is successfully extracted.
+    """
+
+    def __init__(self, profile_name: str, class_name: str) -> None:
+        self.message = "Security request made to IRRSMO00 failed."
+        admin_types = ["USER", "GROUP", "DATASET"]
+        if class_name not in admin_types:
+            self.message += (
+                "\n\nTarget profile "
+                + f"'{profile_name}' already exists as a profile in the {class_name} class."
+            )
+        else:
+            self.message += (
+                "\n\nTarget profile "
+                + f"'{profile_name}' already exists as a {class_name} profile."
```

Comment:
Surround `class_name` with single quotes:

```python
            self.message += (
                "\n\nTarget profile "
                + f"'{profile_name}' already exists as a '{class_name}' profile."
            )
```

Reply:
Agree, that's cleaner. Change made across the board. Corresponding changes also made in unit testing where the error contents are evaluated.